### DISCUSSION TITLE
是否发一次电就一劳永逸了

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1432] [提问] [answered]
就是，是不是瞎选一个档位发一次电就不用每月都交钱了

**Comment by WForst-Breeze** [comment, marked as answer]
并不是，每个档位都有相应的赞助者权益。

6.66 档位仅能获得最新版快照下载地址，但不提供后续快照更新
12.34 档位及以上就可以进入内群并长期获取快照更新啦……

**Reply by WForst-Breeze** [reply]
好像是的（）不过我想表达的是 6.66 档位是一次性的，需要持续赞助，而 12.34 以上就可以只交一次就 OK 了

**Reply by iPodtouch6557** [reply]
> 好像是的（）不过我想表达的是 6.66 档位是一次性的，需要持续赞助，而 12.34 以上就可以只交一次就 OK 了

了解了，马上去发23.33档（）
